homes_guests_loves.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Title</title>
+    <link href="homes_guests_loves_styles.css" rel="stylesheet">
+</head>
+<body>
+<h1 class="col-lg-12 col-sm-8">Homes guests loves</h1>
+
+<!--<svg class="svg" width="296" height="296" viewBox="0 0 296 296" fill="none" xmlns="http://www.w3.org/2000/svg">-->
+
+    <div class="home_photos_box col-lg-10 col-sm-12">
+
+        <div class="home_place_box box1">
```

Comment:
why you need box1 class here? I don't see any styles for it. Same for classes box2, box3, box4